locales/en.default.schema.json

```diff
@@ -1047,7 +1047,7 @@
         "margin_top": {
           "label": "Top margin"
         },
-         "header__9": {
+        "header__9": {
```

Comment:
It would be great if this PR got merged in so that we don't have this kinds of changes anymore: Add prettier config to support format-on-save #2323